### ISSUE TITLE
Win10 GUI "Connecting to Wallet" spinner after deleting active plotting [BUG]

### ISSUE THREAD

**Issue by soundfx42** [Chia-Network/chia-blockchain#1547]
**Describe the bug**
When deleting a plot that is actively plotting the GUI changes to a blank screen except for the text "Connecting to wallet" and a loading spinner. Selecting File -> Exit change screen to shutting down node but never progresses. Have to kill program via task manager .

The arrangement has occurred whilst plotting two plots in parallel with more plots queued. Each of the two parallel plots are being plotted to separate drives. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Under Plots select the three dots under Action next to a plot that is plotting and select delete.

**Expected behavior**
Plot should be deleted with no interruption to other processes 

**Desktop (please complete the following information):**
- OS: Windows 10 Home
- CPU: Intel i7-7700HQ
- RAM: 32 GB

![Screenshot 2021-03-29 084106](https://user-images.githubusercontent.com/7479640/112769005-907d5c80-906a-11eb-9b31-cdc1d1e61ffc.jpg)

**Comment by hoffmang9** [comment]
Duplicate of https://github.com/Chia-Network/chia-blockchain-gui/issues/44

**Comment by adaptdefymike** [comment]
I believe the same thing is happening to me. I had two system crashes while plotting for reasons unknown to me. Then when the plotting stalled I deleted it and now I get the wheel of doom. "connecting to wallet" and nothing else. This was while running on 1.0. I installed 1.01 and still the same. No go. Anyone got a work around?

**Comment by hoofin22** [comment]
The same thing has happened to me - what is required to solve this?

**Comment by MCorluka** [comment]
Similar issue.
Was plotting two plots at the same time, slightly staggered start times. RAM about 4GB/plot, 1 thread, plenty of space. Internet off. Hour 6. Decided to delete one in-progress plot as it's my main computer and both weren't going to finish before bed.  I turned on the internet to quickly check something.
"Connecting to wallet" error happens and never goes away.

What do I need to solve this? This is the 2nd time a "Connecting to Wallet" error happens after I delete an active plot. Would appreciate any help!

**Comment by bo3b** [comment]
Open the developer Tools shows the repeating error, every second:

`events.js:292 Uncaught Error: connect ECONNREFUSED 127.0.0.1:55400
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141)
exceptionWithHostPort @ internal/errors.js:563
afterConnect @ net.js:1141
main.4c627c78.chunk.js:1 Already connected, not reconnecting.
main.4c627c78.chunk.js:1 0
main.4c627c78.chunk.js:1 Already connected, not reconnecting.
main.4c627c78.chunk.js:1 0
`

**Comment by hoangan173** [comment]
![Capture](https://user-images.githubusercontent.com/53848223/114196781-e233ba00-997b-11eb-981b-55f4cbfec975.PNG)
The same issue has happened to me.
I have to kill the process every time.

**Comment by Daeamon** [comment]
Same issue
`chia.exe start all
Daemon not started yet
Starting daemon
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 80, in <module>
  File "chia\cmds\chia.py", line 76, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\chia.py", line 59, in run_daemon_cmd
  File "asyncio\base_events.py", line 587, in run_until_complete
  File "chia\daemon\server.py", line 936, in async_run_daemon
  File "chia\daemon\server.py", line 144, in start
  File "websockets\server.py", line 965, in __await_impl__
  File "asyncio\base_events.py", line 1389, in create_server
OSError: [Errno 10013] error while attempting to bind on address ('::1', 55400, 0, 0): сделана попытка доступа к сокету методом, запрещенным правами доступа
[13004] Failed to execute script chia
Daemon not started yet
Failed to create the chia daemon`

**Comment by hoffmang9** [comment]
If you get `error while attempting to bind on address ('::1', 55400, 0, 0)` you may need to change `localhost` to `127.0.0.1` in all instances in `~/.chia/mainnet/config/config.yaml`

**Comment by philtreF** [comment]
> If you get `error while attempting to bind on address ('::1', 55400, 0, 0)` you may need to change `localhost` to `127.0.0.1` in all instances in `~/.chia/mainnet/config/config.yaml`

I confirm the bug, and changing  `localhost` to `127.0.0.1` did not fix it for me...

**Comment by oxygen** [comment]
Context:

Drive A with 2 plots in queue, running the first.
Drive B with 2 plots in queue, running the first.
Drive C with 2 plots in parallel, both running.
Drive D with 2 plots in parallel, both running.
Drive E with 1 plot running.

Action:
Deleted 2nd plot of drive C. GUI shows "Removing".
Deleted 2nd plot of drive D. GUI shows "Running".
Minimized the app.

Came back after a few minutes: 
 - GUI shows "Connecting to wallet".
 - All plotting progress is lost, RAM is empty, nothing is happening anymore - restarting the app didn't resume plotting.

**Comment by mattlacey** [comment]
Similar setup as @oxygen, figured now that delete was enabled in the GUI it might actually work, but now I'm going to have to restart a bunch of plots :(

**Comment by molo0928** [comment]
Exact same problem here. Attempted to delete plot in progress, went to connecting to wallet with spinning circle. No modifications to plot log files since.

**Comment by janwarming** [comment]
same problem
